mcs/class/System.Drawing/Test/System.Drawing.Drawing2D/GraphicsPathIteratorTest.cs

```diff
@@ -73,85 +73,79 @@ public void NextSubpath_Null ()
 		}
 
 		[Test]
-		[ExpectedException (typeof (NullReferenceException))]
 		public void CopyData_NullPoints ()
 		{
 			using (GraphicsPath gp = new GraphicsPath ()) {
 				gp.AddLines (pts_2f);
 				using (GraphicsPathIterator gpi = new GraphicsPathIterator (gp)) {
 					PointF [] points = null;
 					byte [] types = new byte [1];
-					Assert.AreEqual (0, gpi.CopyData (ref points, ref types, 0, 1));
+					Assert.Throws<NullReferenceException> (() => Assert.AreEqual (0, gpi.CopyData (ref points, ref types, 0, 1)));
```

Comment:
Agreed, the Assert.AreEqual is superfluous (GraphicsPathIterator.CopyData will throw nullref because points is null). I didn't comment on it since it's a literal translation of the original code but we should maybe clean this up while we're modifying the code.